src/includes/getting-started-config/javascript.remix.mdx

```diff
@@ -0,0 +1,120 @@
+To use this SDK, initialize Sentry in your Remix entry points for both the client and server.
+
+```typescript {filename: entry.client.tsx}
+import { useLocation, useMatches } from "@remix-run/react";
+import * as Sentry from "@sentry/remix";
+import { useEffect } from "react";
+
+Sentry.init({
+  dsn: "__DSN__",
+  tracesSampleRate: 1,
+  integrations: [
+    new Sentry.BrowserTracing({
+        routingInstrumentation: Sentry.remixRouterInstrumentation(
+            useEffect,
+            useLocation,
+            useMatches,
+        ),
+    }),
+  ],
+  // ...
+});
+```
+
+Initialize Sentry in your entry point for the server, to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/v1/api/conventions#action) and [`loader`](https://remix.run/docs/en/v1/api/conventions#loader) functions. You can also initialize Sentry's database integrations such as Prisma to get spans for your database calls.
+
+```typescript {filename: entry.server.tsx}
+import { prisma } from "~/db.server";
+
+import * as Sentry from "@sentry/remix";
+
+Sentry.init({
+  dsn: "__DSN__",
+  tracesSampleRate: 1,
+  integrations: [new Sentry.Integrations.Prisma({ client: prisma })],
+  // ...
+});
+```
+
+Also, wrap your Remix root, with Sentry's `ErrorBoundary` component to catch React component errors, and `withSentryRouteTracing` to get parameterized router transactions.
+
+```typescript {filename: root.tsx}
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration,
+} from "@remix-run/react";
+
+import { ErrorBoundary, withSentryRouteTracing } from "@sentry/remix";
```

Comment:
Is `withSentryRouteTracing` exclusive to remix? If so, can we use `ErrorBoundary` inside of the `withSentryRouteTracing`?

This way, users would only have to do `export default withSentryRouteTracing(App);` and that's it.

Reply:
Yes, it's discussed already, but didn't make it into the release. Will open a PR for it 👍

Reply:
ref: https://github.com/getsentry/sentry-javascript/pull/5365

Let's change `withSentryRouteTracing` -> `withSentry`?

Reply:
Thanks!